### PR TITLE
PM-13927: Update primary and secondary button styles

### PR DESCRIPTION
**Pull request by matt-livefront**
## 🎟️ Tracking

[PM-13927](https://bitwarden.atlassian.net/browse/PM-13927)

## 📔 Objective

Updates the button styles to match the updated designs.

This ends up unifying our button styles as well. The transparent and tertiary button styles have both been replaced by the secondary button style.

Sorry for all the snapshot changes 😒. My individual commits don't have snapshot changes, which might be easier to review.

## 📸 Screenshots

Screenshot 2024-12-11 at 11 13 50 AM

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-13927]: https://bitwarden.atlassian.net/browse/PM-13927?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by fedemkr — APPROVED** [review]
Looks great 🎉

File: BitwardenShared/UI/Vault/Vault/VaultGroup/__Snapshots__/VaultGroupViewTests/test_snapshot_empty.1.png
Comment (ezimet-livefront) [inline BitwardenShared/UI/Vault/Vault/VaultGroup/__Snapshots__/VaultGroupViewTests/test_snapshot_empty.1.png:1]: not sure what was the design but is this supposed to take more spaces available?

Comment (matt-livefront) [reply BitwardenShared/UI/Vault/Vault/VaultGroup/__Snapshots__/VaultGroupViewTests/test_snapshot_empty.1.png:1]: No, this is a smaller button in the designs now.

Screenshot 2024-12-12 at 1 31 13 PM

https://www.figma.com/design/eH8ybfmeq726N1SyvcoK2K/Bitwarden-iOS-Mobile-App-Reskin---V3-(current)?node-id=12656-85768&t=L65BzMucACnUu67D-4

Comment (ezimet-livefront) [reply BitwardenShared/UI/Vault/Vault/VaultGroup/__Snapshots__/VaultGroupViewTests/test_snapshot_empty.1.png:1]: nice! looks great then!

**Review by ezimet-livefront — APPROVED** [review]

**Review by KatherineInCode — APPROVED** [review]

**Pull request merged**